omzetten van status RSGB_TOPNL_WAITING naar STAGING_OK werkt niet

Als je een laadproces wilt resetten van `RSGB_TOPNL_WAITING` naar `STAGING_OK` werkt dat niet in versie 1.5.1.

<details><summary>server log / stacktrace:</summary>

```
2017-11-16 10:31:09 WARN  net.sourceforge.stripes.exception.DefaultExceptionHandler#warn - Unhandled exception caught by the Stripes default exception handler.
java.lang.IllegalArgumentException: Unknown name value for enum class nl.b3p.brmo.persistence.staging.LaadProces$STATUS: RSGB_TOPNL_WAITING
	at org.hibernate.type.EnumType.nullSafeGet(EnumType.java:128)
	at org.hibernate.type.CustomType.nullSafeGet(CustomType.java:109)
	at org.hibernate.type.AbstractType.hydrate(AbstractType.java:104)
	at org.hibernate.persister.entity.AbstractEntityPersister.hydrate(AbstractEntityPersister.java:2283)
	at org.hibernate.loader.Loader.loadFromResultSet(Loader.java:1527)
	at org.hibernate.loader.Loader.instanceNotYetLoaded(Loader.java:1455)
	at org.hibernate.loader.Loader.getRow(Loader.java:1355)
	at org.hibernate.loader.Loader.getRowFromResultSet(Loader.java:611)
	at org.hibernate.loader.Loader.doQuery(Loader.java:829)
	at org.hibernate.loader.Loader.doQueryAndInitializeNonLazyCollections(Loader.java:274)
	at org.hibernate.loader.Loader.loadEntity(Loader.java:2037)
	at org.hibernate.loader.entity.AbstractEntityLoader.load(AbstractEntityLoader.java:86)
	at org.hibernate.loader.entity.AbstractEntityLoader.load(AbstractEntityLoader.java:76)
	at org.hibernate.persister.entity.AbstractEntityPersister.load(AbstractEntityPersister.java:3294)
	at org.hibernate.event.def.DefaultLoadEventListener.loadFromDatasource(DefaultLoadEventListener.java:496)
	at org.hibernate.event.def.DefaultLoadEventListener.doLoad(DefaultLoadEventListener.java:477)
	at org.hibernate.event.def.DefaultLoadEventListener.load(DefaultLoadEventListener.java:227)
	at org.hibernate.event.def.DefaultLoadEventListener.proxyOrLoad(DefaultLoadEventListener.java:285)
	at org.hibernate.event.def.DefaultLoadEventListener.onLoad(DefaultLoadEventListener.java:152)
	at org.hibernate.impl.SessionImpl.fireLoad(SessionImpl.java:1090)
	at org.hibernate.impl.SessionImpl.get(SessionImpl.java:1005)
	at org.hibernate.impl.SessionImpl.get(SessionImpl.java:998)
	at org.hibernate.ejb.AbstractEntityManagerImpl.find(AbstractEntityManagerImpl.java:781)
	at org.hibernate.ejb.AbstractEntityManagerImpl.find(AbstractEntityManagerImpl.java:756)
	at nl.b3p.brmo.service.stripes.LaadProcesActionBean.saveRecord(LaadProcesActionBean.java:208)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at net.sourceforge.stripes.controller.DispatcherHelper$6.intercept(DispatcherHelper.java:456)
	at net.sourceforge.stripes.controller.ExecutionContext.proceed(ExecutionContext.java:176)
	at net.sourceforge.stripes.controller.BeforeAfterMethodInterceptor.intercept(BeforeAfterMethodInterceptor.java:113)
	at net.sourceforge.stripes.controller.ExecutionContext.proceed(ExecutionContext.java:173)
	at org.stripesstuff.plugin.waitpage.WaitPageInterceptor.intercept(WaitPageInterceptor.java:175)
	at net.sourceforge.stripes.controller.ExecutionContext.proceed(ExecutionContext.java:173)
	at net.sourceforge.stripes.controller.ExecutionContext.wrap(ExecutionContext.java:86)
	at net.sourceforge.stripes.controller.DispatcherHelper.invokeEventHandler(DispatcherHelper.java:454)
	at net.sourceforge.stripes.controller.DispatcherServlet.invokeEventHandler(DispatcherServlet.java:278)
	at net.sourceforge.stripes.controller.DispatcherServlet.service(DispatcherServlet.java:160)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:729)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:292)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:207)
	at net.sourceforge.stripes.controller.StripesFilter.doFilter(StripesFilter.java:260)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:240)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:207)
	at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:52)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:240)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:207)
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:212)
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:106)
	at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:614)
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:141)
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:79)
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:88)
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:528)
	at org.apache.coyote.http11.AbstractHttp11Processor.process(AbstractHttp11Processor.java:1100)
	at org.apache.coyote.AbstractProtocol$AbstractConnectionHandler.process(AbstractProtocol.java:687)
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1520)
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.run(NioEndpoint.java:1476)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
	at java.lang.Thread.run(Thread.java:748)
Caused by: java.lang.IllegalArgumentException: No enum constant nl.b3p.brmo.persistence.staging.LaadProces.STATUS.RSGB_TOPNL_WAITING
	at java.lang.Enum.valueOf(Enum.java:238)
	at org.hibernate.type.EnumType.nullSafeGet(EnumType.java:125)
	... 62 more
```

</details>

--- a/brmo-loader/src/main/java/nl/b3p/brmo/loader/entity/LaadProces.java
+++ b/brmo-loader/src/main/java/nl/b3p/brmo/loader/entity/LaadProces.java
@@ -18,7 +18,11 @@ public class LaadProces {
     private Date statusDatum;
     private String contactEmail;
 
+    /**
+     * @see nl.b3p.brmo.persistence.staging.LaadProces
+     */
     public static enum STATUS {
+        // als je er bij maakt ook in de andere LaadProces klasse toevoegen
         STAGING_OK, STAGING_NOK, ARCHIVE, STAGING_DUPLICAAT, RSGB_BGT_OK, RSGB_BGT_NOK, RSGB_BGT_WAITING, RSGB_TOPNL_WAITING, RSGB_TOPNL_OK, RSGB_TOPNL_NOK
     };
 

--- a/brmo-persistence/src/main/java/nl/b3p/brmo/persistence/staging/LaadProces.java
+++ b/brmo-persistence/src/main/java/nl/b3p/brmo/persistence/staging/LaadProces.java
@@ -56,9 +56,12 @@ public class LaadProces implements Serializable {
     @ManyToOne
     private AutomatischProces automatischProces;
 
+    /**
+     * @see nl.b3p.brmo.loader.entity.LaadProces
+     */
     public static enum STATUS {
-
-        STAGING_OK, STAGING_NOK, ARCHIVE, STAGING_DUPLICAAT, RSGB_BGT_OK, RSGB_BGT_NOK, RSGB_BGT_WAITING
+        // als je er bij maakt ook in de andere LaadProces klasse toevoegen
+        STAGING_OK, STAGING_NOK, ARCHIVE, STAGING_DUPLICAAT, RSGB_BGT_OK, RSGB_BGT_NOK, RSGB_BGT_WAITING, RSGB_TOPNL_WAITING, RSGB_TOPNL_OK, RSGB_TOPNL_NOK
     };
 
         public LaadProces() {

